src/common/filters/cd-url-formatter.js

```diff
@@ -0,0 +1,6 @@
+export default (url) => {
+  if (url.substring(0, 7) === 'http://' || url.substring(0, 8) === 'https://') {
+    return url;
+  }
+  return `http://${url}`;
```

Comment:
This is for external sites. We don't know if they support https or not. I'd imagine most Dojo sites wouldn't. Any that do would usually redirect from http to https anyway, or even have HSTS so it won't even hit their server over http.